### PR TITLE
V4.15.0

### PR DESCRIPTION
**Pull request by Marc-Roig**
### What does it do?

Describe the technical changes you did.

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Review by joshuaellis — DISMISSED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -9660,17 +8557,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@types/prettier@npm:^2.1.5":
-  version: 2.7.0
-  resolution: "@types/prettier@npm:2.7.0"
-  checksum: 5451430048c139456f14cc4eab8e1fd4d2dde4e0e10dbd3b49b8befa173f0958bf477575a4848bacfee5d42e46c4494dc9f5933fe8bcadf43b862741a7d049ad
-  languageName: node
```
Comment (joshuaellis) [inline yarn.lock:9667]: this is why we have an error, i wish eslint would warn us of this sort of thing.

Comment (Marc-Roig) [reply yarn.lock:9667]: oh, nice catch, will update it tomorrowww

Comment (joshuaellis) [reply yarn.lock:9667]: I have a PR that solves this open FYI 😎

Comment (Marc-Roig) [reply yarn.lock:9667]: which PR? 👀

**Comment by Marc-Roig** [comment]
never deleting the yarn.lock again, I promise 🤩

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**